config/jest/spec/setup.js

```diff
@@ -1,9 +1,11 @@
 import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import '@babel/polyfill';
-import 'whatwg-fetch';
```

Comment:
Please check, if this dependency is still needed somewhere else. Otherwise remove it from package.json

Reply:
yes its still needed in webpack config and poylfills.js